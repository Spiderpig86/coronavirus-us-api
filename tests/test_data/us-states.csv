date,state,fips,cases,deaths
2020-01-21,Washington,53,1,0
2020-01-22,Washington,53,1,0
2020-01-23,Washington,53,1,0
2020-01-24,Illinois,17,1,0
2020-01-24,Washington,53,1,0
2020-01-25,California,06,1,0
2020-01-25,Illinois,17,1,0
2020-01-25,Washington,53,1,0
2020-01-26,Arizona,04,1,0
2020-01-26,California,06,2,0
2020-01-26,Illinois,17,1,0
2020-01-26,Washington,53,1,0
2020-01-27,Arizona,04,1,0
2020-01-27,California,06,2,0
2020-01-27,Illinois,17,1,0
2020-01-27,Washington,53,1,0
2020-01-28,Arizona,04,1,0
2020-01-28,California,06,2,0
2020-01-28,Illinois,17,1,0
2020-01-28,Washington,53,1,0
2020-01-29,Arizona,04,1,0
2020-01-29,California,06,2,0
2020-01-29,Illinois,17,1,0
2020-01-29,Washington,53,1,0
2020-01-30,Arizona,04,1,0
2020-01-30,California,06,2,0
2020-01-30,Illinois,17,2,0
2020-01-30,Washington,53,1,0
2020-01-31,Arizona,04,1,0
2020-01-31,California,06,3,0
2020-01-31,Illinois,17,2,0
2020-01-31,Washington,53,1,0
2020-02-01,Arizona,04,1,0
2020-02-01,California,06,3,0
2020-02-01,Illinois,17,2,0
2020-02-01,Massachusetts,25,1,0
2020-02-01,Washington,53,1,0
2020-02-02,Arizona,04,1,0
2020-02-02,California,06,6,0
2020-02-02,Illinois,17,2,0
2020-02-02,Massachusetts,25,1,0
2020-02-02,Washington,53,1,0