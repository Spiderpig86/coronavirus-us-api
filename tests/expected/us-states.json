
  [
    {
      "id": "US@Washington@53",
      "country": "US",
      "state": "Washington",
      "county": "",
      "fips": "53",
      "timelines": {
        "confirmed": {
          "history": {
            "2020-01-21T00:00:00Z": 1,
            "2020-01-22T00:00:00Z": 1,
            "2020-01-23T00:00:00Z": 1,
            "2020-01-24T00:00:00Z": 1,
            "2020-01-25T00:00:00Z": 1,
            "2020-01-26T00:00:00Z": 1,
            "2020-01-27T00:00:00Z": 1,
            "2020-01-28T00:00:00Z": 1,
            "2020-01-29T00:00:00Z": 1,
            "2020-01-30T00:00:00Z": 1,
            "2020-01-31T00:00:00Z": 1,
            "2020-02-01T00:00:00Z": 1,
            "2020-02-02T00:00:00Z": 1
          },
          "latest": 1
        },
        "deaths": {
          "history": {
            "2020-01-21T00:00:00Z": 0,
            "2020-01-22T00:00:00Z": 0,
            "2020-01-23T00:00:00Z": 0,
            "2020-01-24T00:00:00Z": 0,
            "2020-01-25T00:00:00Z": 0,
            "2020-01-26T00:00:00Z": 0,
            "2020-01-27T00:00:00Z": 0,
            "2020-01-28T00:00:00Z": 0,
            "2020-01-29T00:00:00Z": 0,
            "2020-01-30T00:00:00Z": 0,
            "2020-01-31T00:00:00Z": 0,
            "2020-02-01T00:00:00Z": 0,
            "2020-02-02T00:00:00Z": 0
          },
          "latest": 0
        }
      },
      "last_updated": "2020-05-30T09:19:06Z",
      "latest": {
        "confirmed": 1,
        "deaths": 0
      }
    },
    {
      "id": "US@Illinois@17",
      "country": "US",
      "state": "Illinois",
      "county": "",
      "fips": "17",
      "timelines": {
        "confirmed": {
          "history": {
            "2020-01-24T00:00:00Z": 1,
            "2020-01-25T00:00:00Z": 1,
            "2020-01-26T00:00:00Z": 1,
            "2020-01-27T00:00:00Z": 1,
            "2020-01-28T00:00:00Z": 1,
            "2020-01-29T00:00:00Z": 1,
            "2020-01-30T00:00:00Z": 2,
            "2020-01-31T00:00:00Z": 2,
            "2020-02-01T00:00:00Z": 2,
            "2020-02-02T00:00:00Z": 2
          },
          "latest": 2
        },
        "deaths": {
          "history": {
            "2020-01-24T00:00:00Z": 0,
            "2020-01-25T00:00:00Z": 0,
            "2020-01-26T00:00:00Z": 0,
            "2020-01-27T00:00:00Z": 0,
            "2020-01-28T00:00:00Z": 0,
            "2020-01-29T00:00:00Z": 0,
            "2020-01-30T00:00:00Z": 0,
            "2020-01-31T00:00:00Z": 0,
            "2020-02-01T00:00:00Z": 0,
            "2020-02-02T00:00:00Z": 0
          },
          "latest": 0
        }
      },
      "last_updated": "2020-05-30T09:19:06Z",
      "latest": {
        "confirmed": 2,
        "deaths": 0
      }
    },
    {
      "id": "US@California@06",
      "country": "US",
      "state": "California",
      "county": "",
      "fips": "06",
      "timelines": {
        "confirmed": {
          "history": {
            "2020-01-25T00:00:00Z": 1,
            "2020-01-26T00:00:00Z": 2,
            "2020-01-27T00:00:00Z": 2,
            "2020-01-28T00:00:00Z": 2,
            "2020-01-29T00:00:00Z": 2,
            "2020-01-30T00:00:00Z": 2,
            "2020-01-31T00:00:00Z": 3,
            "2020-02-01T00:00:00Z": 3,
            "2020-02-02T00:00:00Z": 6
          },
          "latest": 6
        },
        "deaths": {
          "history": {
            "2020-01-25T00:00:00Z": 0,
            "2020-01-26T00:00:00Z": 0,
            "2020-01-27T00:00:00Z": 0,
            "2020-01-28T00:00:00Z": 0,
            "2020-01-29T00:00:00Z": 0,
            "2020-01-30T00:00:00Z": 0,
            "2020-01-31T00:00:00Z": 0,
            "2020-02-01T00:00:00Z": 0,
            "2020-02-02T00:00:00Z": 0
          },
          "latest": 0
        }
      },
      "last_updated": "2020-05-30T09:19:06Z",
      "latest": {
        "confirmed": 6,
        "deaths": 0
      }
    },{
      "id": "US@Arizona@04",
      "country": "US",
      "state": "Arizona",
      "county": "",
      "fips": "04",
      "timelines": {
        "confirmed": {
          "history": {
            "2020-01-26T00:00:00Z": 1,
            "2020-01-27T00:00:00Z": 1,
            "2020-01-28T00:00:00Z": 1,
            "2020-01-29T00:00:00Z": 1,
            "2020-01-30T00:00:00Z": 1,
            "2020-01-31T00:00:00Z": 1,
            "2020-02-01T00:00:00Z": 1,
            "2020-02-02T00:00:00Z": 1
          },
          "latest": 1
        },
        "deaths": {
          "history": {
            "2020-01-26T00:00:00Z": 0,
            "2020-01-27T00:00:00Z": 0,
            "2020-01-28T00:00:00Z": 0,
            "2020-01-29T00:00:00Z": 0,
            "2020-01-30T00:00:00Z": 0,
            "2020-01-31T00:00:00Z": 0,
            "2020-02-01T00:00:00Z": 0,
            "2020-02-02T00:00:00Z": 0
          },
          "latest": 0
        }
      },
      "last_updated": "2020-05-30T09:19:06Z",
      "latest": {
        "confirmed": 1,
        "deaths": 0
      }
    },
    {
      "id": "US@Massachusetts@25",
      "country": "US",
      "state": "Massachusetts",
      "county": "",
      "fips": "25",
      "timelines": {
        "confirmed": {
          "history": {
            "2020-02-01T00:00:00Z": 1,
            "2020-02-02T00:00:00Z": 1
          },
          "latest": 1
        },
        "deaths": {
          "history": {
            "2020-02-01T00:00:00Z": 0,
            "2020-02-02T00:00:00Z": 0
          },
          "latest": 0
        }
      },
      "last_updated": "2020-05-30T09:19:06Z",
      "latest": {
        "confirmed": 1,
        "deaths": 0
      }
    }
  ]